src/main/cms/cms_menu_imu.c

```diff
@@ -391,6 +391,73 @@ static CMS_Menu cmsx_menuFilterPerProfile = {
     .entries = cmsx_menuFilterPerProfileEntries,
 };
 
+static uint8_t cmsx_dstPidProfile;
```

Comment:
I think the CMS code for this should be conditional, so that it can be disabled for F3 if we start running out of space again.

See https://github.com/basdelfos/betaflight/blob/e3c9c64c7f240585efc9733f24c59f0b6cd6ca3e/src/main/cms/cms_menu_osd.c#L42 for an example (or potentially rename this into `DISABLE_EXTENDED_CMS_MENU` and use it in both places.